content/getting-started/managing-your-npm-user-account/changing-your-npm-username.mdx

```diff
@@ -3,5 +3,11 @@ title: Changing your npm username
 ---
 import shared from '../../../src/shared.js'
 
-<>To change your npm username, {shared['contact-support'].text}</>.
+It is not currently possible to change your npm username.  You'll need to
+create a new account and migrate the data to the new account manually.
+
+1. Create a [new user account](/creating-a-new-npm-user-account) with your desired username
+2. [Transfer your packages](/transferring-a-package-from-a-user-account-to-another-user-account) to your new account.
+3. If you are a member of any [organizations](/organizations), ask the organization administrator to [invite your new account to the organization](/adding-members-to-your-organization).
+4. Delete your [original account](/deleting-your-npm-user-account). 
```

Comment:
can someone register this account in the future? if so, it seems dangerous to delete your account; if not, it seems like perhaps people should be warned here that this step is permanent.

Reply:
An additional warning makes sense here.